src/main/scala/bitboard/OpaqueBitboard.scala

```diff
@@ -52,6 +52,20 @@ trait OpaqueBitboard[A](using A =:= Long) extends TotalWrapper[A, Long]:
     // remove the first non empty position
     def removeFirst: A = (a.value & (a.value - 1L)).bb
 
+    // check if the intersection of all bitboards is not empty
+    def sharedAny(o: Long, xs: Long*): Boolean =
```

Comment:
`hasOverlap` perhaps?

Reply:
maybe, or just `overlaps`? imho it goes better with `intersects`.